src-web/scss/ansible-modal.scss

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2020 Red Hat, Inc. */
+/* Copyright Contributors to the Open Cluster Management project */
+
```

Comment:
these are all global selector. it will also be applied on other pages e.g. 
<img width="1557" alt="image" src="https://user-images.githubusercontent.com/16092291/118990865-41102a00-b951-11eb-8d0a-5abe461aa205.png">